pom.xml

```diff
@@ -115,7 +115,8 @@
     <module>google-cloud-pubsub</module>
     <module>google-cloud-resourcemanager</module>
     <module>google-cloud-spanner</module>
-    <module>google-cloud-speech</module>
+    <module>google-cloud-speech-v1</module>
+    <module>google-cloud-speech-v1beta1</module>
```

Comment:
After the second thought, I think merging two versions into one single artifact might be okay. The two versions will stay in separate packages and the users won't notice any differences. Will do.